Apollo.xcodeproj/xcshareddata/xcschemes/Apollo.xcscheme

```diff
@@ -75,9 +75,6 @@
                ReferencedContainer = "container:Apollo.xcodeproj">
             </BuildableReference>
             <SkippedTests>
-               <Test
-                  Identifier = "SQLiteFetchQueryTests">
-               </Test>
```

Comment:
Hmmm, that makes me wonder why there are `SQLiteFetchQueryTests` in the `Apollo` scheme in the first place. I'll see if there's a way to avoid that.

Reply:
It lists the tests that are skipped rather than the tests that run for the cache-dependent test bundle. 

It's super counterintuitive, but having `SQLite` tests listed in here _is_ what you want.